.golangci.yml

```diff
@@ -1,5 +1,11 @@
 ---
 run:
+  build-tags:
+    - apparmor
```

Comment:
What happens if the distribution does not support the build-tag?

Reply:
I guess nothing, but let's see if travis complains (should have no apparmor support).

Reply:
On travis it works seamless:
https://travis-ci.com/cri-o/cri-o/jobs/202074032#L348

I also tested it on a modified Dockerfile without any apparmor support and this works too. :+1: